util/translate_timeline.js

```diff
@@ -87,16 +92,40 @@ const run = async (args) => {
     let line = timelineLines[i].trim();
 
     const lineText = lineToText[lineNumber];
-    if (lineText)
-      line = line.replace(` "${lineText.name}"`, ` "${lineText.text}"`);
-    const lineSync = lineToSync[lineNumber];
-    if (lineSync)
-      line = line.replace(`sync /${lineSync.origRegexStr}/`, `sync /${lineSync.regex.source}/`);
-
-    if (syncErrors[lineNumber])
-      line += ' #MISSINGSYNC';
-    if (textErrors[lineNumber])
-      line += ' #MISSINGTEXT';
+
+    if (args?.colorize === 'true') {
+      if (lineText)
+        line = line.replace(` "${lineText.name}"`, ` \x1b[93m"${lineText.text}"\x1b[0m`);
+      const lineSync = lineToSync[lineNumber];
+      if (lineSync)
+        line = line.replace(`sync /${lineSync.origRegexStr}/`, `\x1b[31msync\x1b[0m \x1b[32m/${lineSync.regex.source}/\x1b[0m`);
+
+      // if a # is in the line, dont make uneccassary colorizatiosn
+      if (line.includes('#')) {
+        line = line.replace('#', `\x1b[90m#`) + '\x1b[0m';
+      } else {
+        line = line.replace(/ window (\d+(,\d+)?)/, `\x1b[31m window\x1b[0m \x1b[35m$1\x1b[0m`);
+        line = line.replace(/ duration (\d+(\.\d+)?)/, `\x1b[31m duration\x1b[0m \x1b[35m$1\x1b[0m`);
+        line = line.replace(/ jump (\d+(\.\d+)?)/, `\x1b[31m jump\x1b[0m \x1b[35m$1\x1b[0m`);
+      }
+      // colorize the lineText.time numbers
+      line = line.replace(/^(\d+\.\d+)/, `\x1b[35m$1\x1b[0m`);
```

Comment:
Honestly it's this section that feels like the most re-parsing.  Would coloring the text, sync, and MISSINGTEXT/MISSINGSYNC be enough (and this block could be removed)?

I can see in this section that:
* if a line has a comment in a text, it will colorize incorrectly
* if a line has a comment, then window/duration/jump don't get colorized
* if a line has a number inside of a text or a comment it will colorize that too

Reply:
problem with comments are, that commented sync lines are not translated and could lead to confusion e.g. `#sync /:Lamebrix Strikebocks:1AB[89AB]:/` from a10s

the number inside text should not happen as the last line-.replace matches on line start as it includes `^`.
But the other issues are there, thats right

Reply:
> problem with comments are, that commented sync lines are not translated and could lead to confusion e.g. `#sync /:Lamebrix Strikebocks:1AB[89AB]:/` from a10s

Yeah, that's part of the reason why I was wondering if there was some better way to handle this whole problem, e.g. printing only the texts or only the syncs and not everything else.